server/live_query/live_query_test.go

```diff
@@ -133,9 +134,11 @@ func testLiveQueryOnlyExpired(t *testing.T, store fleet.LiveQueryStore) {
 	require.NoError(t, err)
 	assert.Len(t, queries, 0)
 
-	activeNames, err := redigo.Strings(conn.Do("SMEMBERS", activeQueriesKey))
```

Comment:
the cleanup operation is now running asynchronously so it doesn't delay cache retrieval